cluster/bin/snappy-job.sh

```diff
@@ -119,6 +120,15 @@ while (( "$#" )); do
       shift
       contextName="${1:-$TOK_EMPTY}"
     ;;
+    --user)
+      shift
+      username="${1:-$TOK_EMPTY}"
+      while [ -z ${password} ]; do
+        read -s -p "Enter password for ${username}: " password
+        echo ""
+      done
+      securePart="-u ${username}:${password}"
+    ;;
```

Comment:
curl, on some platforms, claims to hide the credentials sent via -u option.
Nevertheless, will change to another option from curl, --netrc, which takes the credentials in a netrc file with a simple format. So --user will be removed from above and replaced with --passfile.
Will file JIRAs for the generic requirement, anyway.